pkg/kubectl/describe.go

```diff
@@ -477,8 +477,9 @@ func (d *ServiceDescriber) Describe(namespace, name string) (string, error) {
 
 	endpoints, _ := d.Endpoints(namespace).Get(name)
 	events, _ := d.Events(namespace).Search(service)
+	pods, _ := d.Pods(namespace).List(labels.Set(service.Spec.Selector).AsSelector(), fields.Everything())
```

Comment:
This isn't going to get the set you want, I don't think. It will include terminated and not-ready pods.